wagtail/admin/views/generic/mixins.py

```diff
@@ -129,12 +129,14 @@ def get_context_data(self, **kwargs):
 
 
 class PanelMixin:
```

Comment:
This should really be `EditHandlerMixin` if we want to keep the naming of "edit handler" for the top-level/root panel.

This mixin was made around the time the `EditHandler` class was renamed to `Panel`. I assumed we would completely drop the "edit handler" term, hence I named it `PanelMixin`.

Reply:
Either name works for me really - it's the mixin that provides a top-level edit handler property, or the one that enables panel-based editing as a feature.